include/vkgcDefs.h

```diff
@@ -266,6 +267,7 @@ struct PipelineOptions {
 
   ShadowDescriptorTableUsage shadowDescriptorTableUsage; ///< Controls shadow descriptor table.
   unsigned shadowDescriptorTablePtrHigh;                 ///< Sets high part of VA ptr for shadow descriptor table.
+  bool robustAccessEx;                                   ///< If set, enable workaround to support VK_EXT_robustness2
```

Comment:
I don't think I would call this a "workaround".

Reply:
How about just support VK_EXT_robustness2?

Reply:
Yes sounds good.